docs/admin/auth/keycloak.md

```diff
@@ -0,0 +1,7 @@
+# Keycloak Authentication with Coder
+
+The access_type parameter has two possible values: "online" and "offline." By default, the value is set to "offline". This means that when a user authenticates using OIDC, the application requests offline access to the user's resources, including the ability to refresh access tokens without requiring the user to reauthenticate.
+
+To enable the `offline_access` scope, which allows for the refresh token functionality, you need to add it to the list of requested scopes during the authentication flow. Including the `offline_access` scope in the requested scopes ensures that the user is granted the necessary permissions to obtain refresh tokens.
```

Comment:
Should we provide the example?

```console
CODER_OIDC_SCOPES=openid,profile,email,offline_access
```

Reply:
It would be better if we also include an example